Toolset/palettes/standalone settings/revstandalonesettingsiosbehavior.livecodescript

```diff
@@ -419,13 +434,29 @@ end chooseImageFileWithDimension
 
 on tipDisplayUpdate
    if the visible of group "iOSCommercialTip" then
-      set the topLeft of group "iosBuild" to 12,136
-      set the topLeft of group "iOS Settings" to -1,136
-      set the cSize of this card to 1024,726
+      set the topLeft of group "buildbutton" to 50,127
+      set the topLeft of group "iosSettings" to 5,127
```

Comment:
You should change the name of the group from `"iosSettings"` to (the old) `"iOS Settings"`, since there are existing references to "iOS Settings". This will fix the following bug:
1. Open the iOS Standalone Settings on Windows
Expected result: The ios settings be disabled
Observed result: They are not
PS: Make sure you change the name in both:
- The scripts that refer to it
- The group name itself (using e.g the Property Inspector for that group)